cmd/config.go

```
@@ -215,10 +303,26 @@ func writeFile(filename string, data []byte, perm os.FileMode) error {
 	return err
 }
 
+func readKeyFromFile(filepath string) (*keymgmt.NACLKey, error) {
+	keyBytes, err := ioutil.ReadFile(filepath)
+	if err != nil {
+		panic(err)
```

Why do you use `panic(err)` instead of `return nil, err` here?